startup.d/40-run-puppet.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
 cd puppet
+/bin/bash pullForge.sh
```

Comment:
`./pullForge.sh` should do the same, since `pullForge.sh` has `#!/bin/bash` at the top.